server/system/types/user.go

```diff
@@ -163,3 +164,13 @@ func (u *User) Clone() *User {
 
 func (meta *UserMeta) Scan(src any) error           { return sql.ParseJSON(src, meta) }
 func (meta *UserMeta) Value() (driver.Value, error) { return json.Marshal(meta) }
+
+func ParseUserMeta(ss []string) (p *UserMeta, err error) {
+	p = &UserMeta{}
+
+	if len(ss) == 0 {
+		return
+	}
+
+	return p, json.Unmarshal([]byte(ss[0]), p)
```

Comment:
```suggestion
	return p, json.Unmarshal([]byte(ss[0]), &p)
```

Shouldn't the Unmarshal fnc get the pointer?